lighthouse-core/config/config.js

```diff
@@ -568,6 +569,11 @@ class Config {
   get categories() {
     return this._categories;
   }
+
+  /** @type {Object<{title: string, description: string}>} */
```

Comment:
it's an object where the values are `{title: string, description: string}` but IIRC we actually need `Object<string,{title: string, description: string}>` now, will update